editor/src/components/canvas/canvas-strategies/canvas-strategies.tsx

```diff
@@ -91,6 +92,13 @@ const resizeStrategies: MetaCanvasStrategy = (
     [keyboardAbsoluteResizeStrategy, absoluteResizeBoundingBoxStrategy, flexResizeBasicStrategy],
   )
 }
+const flexResizeMetaStrategy: MetaCanvasStrategy = (
+  canvasState: InteractionCanvasState,
+  interactionSession: InteractionSession | null,
+  customStrategyState: CustomStrategyState,
+): Array<CanvasStrategy> => {
+  return mapDropNulls((factory) => factory(canvasState, interactionSession), [flexResizeStrategy])
```

Comment:
Was the basic flex resize strategy supposed to be moved here too?

Reply:
Not for now, this is only needed to be able to use a smaller set of available strategies for insertion. I hope it's here only temporarily, I added a comment.